draft-ietf-mmusic-dtls-sdp.xml

```diff
@@ -368,10 +386,12 @@
             <t>
                 If the offerer inserts the SDP 'setup' attribute with an 'actpass' or 'passive' attribute value, the
                 offerer MUST be prepared to receive a DTLS ClientHello message (if a new DTLS association
-                is established by the answerer) from the answerer before the offerer receives the SDP answer.
+                is established by the answerer) from the answerer before the offerer receives the SDP answer. In such
+                case the offerer MAY cache the ClientHello message, but the offerer MUST NOT initiate a DTLS

```

Comment:
What would be the purpose to start the DTLS handshake before the answer is received? This situation should only occur as a race condition between the answer and ClientHello, so any additional delay would be minimal.

As far as regular UDP is concerned, end point technically does not know the remote address where date is supposed to be sent until the answer is received. Nothing dictates that UDP based connection is symmetric. Only place where this legitimately works is offer from ICE-lite endpoint, which would send response to ClientHello to the address derived from ICE connectivity check.

If this change is acceptable, I will go through the document and clean up any remaining references to early handshake.